### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by maybenikhil**
Removed the supported hardware area (linked to the wiki instead, for size) and added some build flags.

### CODE REVIEW

**Comment by maybenikhil** [comment]
also, I can add a contributors list (like BF), but I would need a full list

File: README.md [hunk 1]
```diff
@@ -49,10 +51,12 @@ Links:
 - [20x20 900MHz RX](https://github.com/AlessandroAU/ExpressLRS/tree/master-dev/PCB/900MHz/RX_20x20_0603_SMD) Convenient Stack Mounted DIY 20x20mm 868/915MHz RX
 - [20x20 900MHz RX](https://github.com/AlessandroAU/ExpressLRS/tree/master-dev/PCB/900MHz/RX_20x20_0805_SMD) Convenient Stack Mounted DIY 20x20mm 868/915MHz RX
 
-## Long Range Leaderboard
-One of the most frequently asked questions that gets asked from people who are interested in, but haven't yet tried ELRS is "How far does it go, and at what packet rate?"
+## Long Range Competion
```
Comment (JBKingdon) [inline README.md:54]: 'Competion' - typo

Comment (maybenikhil) [reply README.md:54]: whoops

**Comment by AlessandroAU** [comment]
I don't know about removing the the table of hardware, I think the readme should allow someone to decide if this project is for them and the supported hardware is integral to that, I think. What do others think?

**Comment by maybenikhil** [comment]
Understood. If enough people agree I will spend the time making a better/stylized hardware support list for the readme

**Comment by maybenikhil** [comment]
finished!

**Comment by maybenikhil** [comment]
slimmed down the hardware support list and added a link to the wiki which removed some of the repetition in it, without actually losing information

**Comment by maybenikhil** [comment]
replying to this to bump it

**Comment by maybenikhil** [comment]
should i close this?

**Comment by AlessandroAU** [comment]
Sorry, we were lazy... lets get it merged. I'll look at it tonight.

**Pull request merged**